ansible/roles/ros2/tasks/main.yaml

```
@@ -57,6 +57,17 @@
     state: latest
     update_cache: true
 
+- name: Install ros2 dependencies
+  become: true
+  ansible.builtin.apt:
+    name:
+      - ros-humble-camera-info-manager
```

It would be nice to use a distribution-control variable rather than specifying directly.
```suggestion
      - ros-{{ rosdistro }}-camera-info-manager
```